### ISSUE TITLE
[Feature] add leather lost mc cuts the originals

### ISSUE THREAD

**Issue by gunnergunnerson** [FrenzyRoleplay/server-issues#696]
would like to have the original lost mc cuts from the base gta game

**Comment by ProximateTv** [comment]
I don't believe the leather Lost MC vests were ever removed, but the base multiplayer clothing does not include Lost MC vests so I am not 100% certain to which vests you are referring

**Comment by Nussifer** [comment]
the lost vest are in city still

**Comment by gunnergunnerson** [comment]
ok would it be a problem to get these? https://www.gta5-mods.com/player/lost-mc-and-angels-of-death-biker-vests

**Comment by ProximateTv** [comment]
That is just a different texture skinning with different badges for the vests that are currently in the city. If you want leather vs jean vests and you want it to be monochromatic vs in color I can make those changes to the current vests easily enough

**Comment by gunnergunnerson** [comment]
no i for sure want leather cuts but is there anyway you can make them blaine county lost cuts and add 1% cuts please

**Comment by ProximateTv** [comment]
Sure I can change it to Blaine County. The current vests already have a 1% patch on them so I'm not sure what you mean

**Comment by gunnergunnerson** [comment]
ok i didnt see a leather lost vest with the 1% ill look again, and if we could get the cuts changed to vest instead of jackets please

**Comment by gunnergunnerson** [comment]
firecrack can explain better then me.  im sorry

**Comment by FirecrackThat** [comment]
Jacket 245 and 246 are exactly what we're looking for just with the bottom rocker changed to blaine county and the 1% patch added.

**Comment by ProximateTv** [comment]
There is no rocker on those vests. I can change the current vest under vests to leather and Blaine county with relative ease. Anything else is going to be a much longer project

**Comment by FirecrackThat** [comment]
Okay let's do that then. We just don't want to eliminate the denim one entirely because we're not associated with the people currently wearing it.

**Comment by FirecrackThat** [comment]
Also monochromatic like you said would be appreciated! thanks for your patience.
